readme.txt

```diff
@@ -68,6 +68,29 @@ In any case, passing the checks in this tool likely helps to achieve a smooth pl
 
 == Changelog ==
 
+= 1.2.0 =
+
+- The Plugin Check is now part of the automatic checks for plugin submissions on WordPress.org.
```

Comment:
```suggestion
* The Plugin Check is now part of the automatic checks for plugin submissions on WordPress.org.
```

Reply:
This line is not reallly a change in the plugin itself though, so maybe it could also be removed entirely?

Reply:
Agree with @swissspidy

Reply:
ok